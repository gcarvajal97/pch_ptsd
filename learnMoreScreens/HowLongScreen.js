import i18n from 'i18n-js';
import React, { Component } from "react";
import { View, StyleSheet, Text, ScrollView, Dimensions} from "react-native";
import { Video } from 'expo-av';
import { Linking } from "expo";
import translate, { getVideo } from '../components/translateService';
import colors from '../colors';

const videos = {
    en: require("../assets/videos/vidHowLong.mp4"),
    es: require("../assets/videos/vidHowLongEs.mp4"),
};

// Tested and works on iOS
// Tested and works on Android  

class HowLongScreen extends Component {

    // Gives the NavBar a title for this specific screen
    static navigationOptions = {
        headerTitle: translate('howLong.header')
    }

    render() {
        return (
            // renders the learn more screen
            <View style={styles.container}>
                <Text style={styles.title}>
                    {translate('howLong.title')}
                </Text>
                <Video
                    source={getVideo(videos)}
                    rate={1.0}
                    volume={1.0}
                    resizeMode={Video.RESIZE_MODE_STRETCH}
                    resizeMode={Video.RESIZE_MODE_COVER}
                    shouldPlay
                    useNativeControls
                    style={{ 
                        height: 204, 
                        width: '100%',
                        maxHeight: 254, 
                        borderColor: colors.secondary.pms_3005, 
                        borderWidth: 14, 
                        borderTopWidth: 0, 
                        borderBottomWidth: 0, 
                        marginBottom:-1
                    }}
                    accessible
                    accessibilityLabel={translate(
                        'howLong.content.videoCard.accessibility'
                        )}
                    accessibilityHint={translate(
                        'howLong.content.videoCard.accessibility'
                        )}
                />
                <Text style={styles.subTitle}>
                    <Text style={{ fontWeight: 'bold' }}>{translate('howLong.content.videoCard.title')}</Text>
                    {translate('howLong.content.videoCard.subtitle')}
                </Text>
                <ScrollView>
                    <Text style={styles.paragraph}>{translate('howLong.content.mainParagraphOne')}</Text>
                    <Text style={styles.paragraph}>{translate('howLong.content.mainParagraphTwo')}</Text>
                    <Text style={styles.paragraph}>{translate('howLong.content.mainParagraphThree')}</Text>
                    <Text style={styles.paragraphTitle}>{translate('howLong.content.concernParagraph.mainTitle')}</Text>
                    <Text style={styles.paragraph}>{translate('howLong.content.concernParagraph.paragraphOne')}</Text>
                    <Text style={styles.paragraphTitle}>{translate('howLong.content.concernParagraph.bulletList.title')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.concernParagraph.bulletList.bulletOne')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.concernParagraph.bulletList.bulletTwo')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.concernParagraph.bulletList.bulletThree')}</Text>
                    <Text style={styles.paragraphTitle}>
                    {translate('howLong.content.whatIsPTSDParagraph.title')}
                    </Text>
                    <Text style={styles.paragraph}>
                    {translate('howLong.content.whatIsPTSDParagraph.bulletListOne.title')}
                    </Text>
                    <Text style={styles.bullet}>{translate('howLong.content.whatIsPTSDParagraph.bulletListOne.bulletOne')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.whatIsPTSDParagraph.bulletListOne.bulletTwo')}</Text>
                    <Text style={styles.paragraph}>
                    {translate('howLong.content.whatIsPTSDParagraph.paragraphOne')}
                    </Text>
                    <Text style={styles.paragraph}>
                    {translate('howLong.content.whatIsPTSDParagraph.paragraphTwo')}
                    </Text>
                    <Text style={{fontWeight: 'bold', fontSize:16, alignSelf:'stretch', textAlign:'left',marginLeft:5, marginTop:4, marginBottom:3}}>
                    {translate('howLong.content.whatIsPTSDParagraph.bulletListTwo.title')}
                    </Text>
                    <Text style={styles.bullet}>{translate('howLong.content.whatIsPTSDParagraph.bulletListTwo.bulletOne')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.whatIsPTSDParagraph.bulletListTwo.bulletTwo')}</Text>
                    <Text style={styles.bullet}>{translate('howLong.content.whatIsPTSDParagraph.bulletListTwo.bulletThree')}</Text>
                    <Text 
                        testID='hyperlink'
                        style={styles.link} 
                        onPress={()=> Linking.openURL('https://medlineplus.gov/ency/article/000925.htm')}
                        accessible
                        accessibilityLabel={translate('howLong.content.whatIsPTSDParagraph.hyperLinkOne.accessibilityHint')}
                        accessibilityHint={translate('howLong.content.whatIsPTSDParagraph.hyperLinkOne.accessibilityHint')}>
                        {translate('howLong.content.whatIsPTSDParagraph.hyperLinkOne.text')}
                    </Text>
                    <Text style={styles.paragraph}>
                    {translate('howLong.content.whatIsPTSDParagraph.paragraphThree')}
                    </Text>
                </ScrollView>
        </View>
        );
    }
}
export default HowLongScreen;

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        margin: 10,
        backgroundColor: colors.primary.white,
        shadowColor: "#000",
        shadowOffset: {
            width: 0,
            height: 2,
        },
        shadowOpacity: 0.23,
        shadowRadius: 2.62,

        elevation: 4,
    },
    title: {
        backgroundColor: colors.secondary.pms_3005, 
        color: colors.primary.white, 
        fontWeight: 'bold', 
        fontSize: 22, 
        textAlign: 'center', 
        alignSelf: 'stretch'
    },
    subTitle: {
        backgroundColor: colors.secondary.pms_3005, 
        color: colors.primary.white, 
        alignSelf: 'stretch', 
        paddingLeft: 15, 
        paddingRight: 15, 
        fontSize: 15, 
        borderBottomWidth: 2, 
        borderBottomColor: colors.secondary.pms_3005,
    },
    paragraphTitle: {
        fontWeight: 'bold',
        fontSize: 20,
        alignSelf: 'stretch',
        textAlign: 'left',
        marginLeft: 5,
        marginTop: 4,
        marginBottom: 3,
        color: colors.secondary.pms_3005
    },
    bullet: {
        marginVertical: 1, 
        marginHorizontal: 18, 
        textAlign: 'left', 
        alignSelf: 'stretch', 
        fontWeight: 'bold',
        fontSize: 14
    },
    paragraph: {
        alignSelf: "center", 
        paddingVertical: 1, 
        paddingHorizontal: 5,
        fontSize: 14,
        marginTop: 5,
    },
    link: {
        alignSelf:"center", 
        paddingVertical: 1, 
        paddingHorizontal: 5,
        fontSize: 16,
        color: colors.secondary.pms_3005,
        textDecorationLine: 'underline',
        marginTop: 4,
        marginTop: 3,
    }
});