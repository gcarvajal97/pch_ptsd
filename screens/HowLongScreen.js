import React, { Component } from "react";
import { View, StyleSheet, Text, ScrollView, Dimensions} from "react-native";
import { Video } from 'expo-av';
import { Linking } from "expo";

class HowLongScreen extends Component {

    // Gives the NavBar a title for this specific screen
    static navigationOptions = {
        headerTitle: 'How Long Do Reactions Last'
    }

    render() {
        return (
            // renders the learn more screen
            <View style={styles.container}>
                <Text style={styles.title}>
                    How Long Do Traumatic Stress Reactions Last?
                </Text>
                <Video
                    source={require('../assets/videos/vidHowLong.mp4')}
                    rate={1.0}
                    volume={1.0}
                    resizeMode={Video.RESIZE_MODE_STRETCH}
                    resizeMode={Video.RESIZE_MODE_COVER}
                    shouldPlay
                    useNativeControls
                    style={{ height: undefined, width: Dimensions.get('window').width, flex: 1, maxHeight: 254 }}
                    style={{ height: 208, width: '100%', maxHeight: 254, borderColor: '#2089DC', borderWidth: 12, borderTopWidth: 0, borderBottomWidth: 12, marginBottom:-1}}
                />
                <ScrollView>
                    <Text style={styles.paragraph}>
                    {"\n"}
                    {'\t'} For many children and their parents early traumatic stress reactions get better over the first month. 
                    But about 1 in 6 still have traumatic stress reactions that bother them, even 6 months after an injury. 
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    {'\t'} Nearly everyone feels upset or anxious early on. 
                    Over the next few weeks and months, these reactions usually get weaker and then go away. 
                    Even children recovering from serious injuries start to develop ways to cope.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    {'\t'} But for some people, these early reactions do not seem to get better as the months go by, or they may get worse. 
                    Even if the physical injury has healed, they may not have recovered from the trauma. 
                    The intensity and length of the traumatic stress reactions is not related to how bad the physical injury was.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraphTitle}>
                    Why be concerned?
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    When traumatic stress reactions last for weeks or months, they can get in the way of getting back to normal and feeling like yourself again.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraphTitle}>
                    For children and teens, this can affect school, home, and play.
                    {"\n"}
                    </Text>
                    <Text style={styles.bullet}>- Schoolwork and learning might be disrupted because a child cannot concentrate or sleep well.</Text>
                    <Text style={styles.bullet}>- Family relationships and friendships might suffer.</Text>
                    <Text style={styles.bullet}>- Children might stop doing things that they enjoy, or stop trying new things.</Text>
                    <Text style={styles.paragraphTitle}>
                    {"\n"}
                    What is posttraumatic stress disorder, or 'PTSD'?
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    Posttraumatic stress disorder (PTSD) is the name given to traumatic stress reactions that:
                    {"\n"}
                    </Text>
                    <Text style={styles.bullet}>- Are so severe that they get in the way of normal life and</Text>
                    <Text style={styles.bullet}>- Last for more than one month.</Text>
                    <Text style={styles.paragraph}>
                    {"\n"}
                    It is important to know if your child or someone else in your family develops posttraumatic stress disorder from the accident or injury. It is much more common than people realize.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    Up to 1 in 10 people develop posttraumatic stress disorder at some point in their lives.
                    {"\n"}
                    </Text>
                    <Text style={{fontWeight: 'bold', fontSize:20, alignSelf:'stretch', textAlign:'left',marginLeft:5, marginTop:2, marginBottom:1}}>
                    After an injury, posttraumatic stress disorder can:
                    {"\n"}
                    </Text>
                    <Text style={styles.bullet}>- Get in the way of physical recovery</Text>
                    <Text style={styles.bullet}>- Contribute to new medical problems; or</Text>
                    <Text style={styles.bullet}>- Just make it harder to enjoy life.</Text>
                    <Text style={styles.link} onPress={()=> Linking.openURL('https://medlineplus.gov/ency/article/000925.htm')}>
                    {"\n"}
                    Click here for more information on PTSD.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    The good news is that very good treatments for PTSD are available. 
                    Unfortunately, less than half of people with traumatic stress symptoms talk to anyone about their problems or get any professional help, even when their symptoms go on for a long time.
                    {"\n"}
                    </Text>
                    <Text style={styles.paragraph}>
                    This app will help you know when to reach out for help if you need it.
                    {"\n"}
                    </Text>
                    <Text style={{paddingBottom: 230}}></Text>
                </ScrollView>

        </View>
        );
    }
}
export default HowLongScreen;

const styles = StyleSheet.create({
    title: {
        backgroundColor: '#2089DC', 
        color: 'white', 
        fontWeight: 'bold', 
        fontSize: 22, 
        textAlign: 'center', 
        alignSelf: 'stretch'
    },
    paragraphTitle: {
        fontWeight: 'bold',
        fontSize:20,
        alignSelf:'stretch',
        textAlign:'left',
        marginLeft:5,
        marginTop:2,
        marginBottom:1,
        color:'#2089DC'
    },
    bullet: {
        marginVertical:1, 
        marginHorizontal:18, 
        textAlign:'left', 
        alignSelf:'stretch', 
        fontWeight:'bold',
        fontSize:16
    },
    paragraph: {
        alignSelf:"center", 
        paddingVertical:1, 
        paddingHorizontal:5,
        fontSize: 16
    },
    link: {
        alignSelf:"center", 
        paddingVertical:1, 
        paddingHorizontal:5,
        fontSize: 16,
        color:'#2089DC',
        textDecorationLine: 'underline'
    }
});