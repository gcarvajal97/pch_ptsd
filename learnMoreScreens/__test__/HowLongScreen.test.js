import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from 'react';
import HowLongScreen from '../HowLongScreen';
import renderer from 'react-test-renderer';
import {Linking} from 'expo';

configure({ adapter: new Adapter() });
jest.mock('expo', () => ({
    Linking: {
        openURL: jest.fn()
    }
}));


describe("HowLongScreen", () => {
    afterEach(() => {
        jest.clearAllMocks();
      });

    it("renders correctly", () => {
        expect(renderer.create(<HowLongScreen />).toJSON()).toMatchSnapshot();
    });

    it("clicks external link, opens correct website", () => {
        const wrapper = shallow(<HowLongScreen />);
        wrapper.findWhere((node) => node.prop('testID') === 'hyperlink').props().onPress();
        expect(Linking.openURL).toHaveBeenCalledWith('https://medlineplus.gov/ency/article/000925.htm');
    })
})