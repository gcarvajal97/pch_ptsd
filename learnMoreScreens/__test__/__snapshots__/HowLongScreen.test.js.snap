// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowLongScreen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "color": "#FFFFFF",
        "fontFamily": "avenir-heavy",
        "fontSize": 22,
        "textAlign": "center",
      }
    }
  >
    How Long Do Traumatic Stress Reactions Last
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#0077C8",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video on how long stress reactions last"
      accessibilityLabel="Video on how long stress reactions last"
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "borderBottomColor": "#0077C8",
        "borderBottomWidth": 2,
        "color": "#FFFFFF",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "avenir-heavy",
        }
      }
    >
      Stress reactions can get in the way. 
    </Text>
    Professional help is important to get back to normal.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        For many children and their parents early traumatic stress reactions get better over the first month. But about 1 in 6 still have traumatic stress reactions that bother them, even 6 months after an injury.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Nearly everyone feels upset or anxious early on. Over the next few weeks and months, these reactions usually get weaker and then go away. Even children recovering from serious injuries start to develop ways to cope.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        But for some people, these early reactions do not seem to get better as the months go by, or they may get worse. Even if the physical injury has healed, they may not have recovered from the trauma. The intensity and length of the traumatic stress reactions is not related to how bad the physical injury was.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Why be concerned?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        When traumatic stress reactions last for weeks or months, they can get in the way of getting back to normal and feeling like yourself again.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        For children and teens, this can affect school, home, and play.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Schoolwork and learning might be disrupted because a child cannot concentrate or sleep well.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Family relationships and friendships might suffer.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Children might stop doing things that they enjoy, or stop trying new things.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        What is post-traumatic stress disorder, or 'PTSD'?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Post-traumatic stress disorder (PTSD) is the name given to traumatic stress reactions that:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Are so severe that they get in the way of normal life
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Last for more than one month.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        It is important to know if your child or someone else in your family develops post-traumatic stress disorder from the accident or injury. It is much more common than people realize.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Up to 1 in 10 people develop post-traumatic stress disorder at some point in their lives.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 16,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        After an injury, post-traumatic stress disorder can:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Get in the way of physical recovery
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Contribute to new medical problems
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Just make it harder to enjoy life.
      </Text>
      <Text
        accessibilityHint="Navigates to a website on the casuses of PTSD"
        accessibilityLabel="Navigates to a website on the casuses of PTSD"
        accessible={true}
        onPress={[Function]}
        style={
          Object {
            "alignSelf": "center",
            "color": "#0077C8",
            "fontSize": 16,
            "marginTop": 3,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
            "textDecorationLine": "underline",
          }
        }
        testID="hyperlink"
      >
        Click here for more information on PTSD.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        The good news is that very good treatments for PTSD are available. Unfortunately, less than half of people with traumatic stress symptoms talk to anyone about their problems or get any professional help, even when their symptoms go on for a long time.
      </Text>
    </View>
  </RCTScrollView>
</View>
`;

exports[`HowLongScreen renders correctly in Spanish 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "elevation": 4,
      "flex": 1,
      "margin": 10,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.23,
      "shadowRadius": 2.62,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "color": "#FFFFFF",
        "fontFamily": "avenir-heavy",
        "fontSize": 22,
        "textAlign": "center",
      }
    }
  >
    ¿Cuánto duran las reacciones de estrés traumático?
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "borderBottomWidth": 0,
        "borderColor": "#0077C8",
        "borderTopWidth": 0,
        "borderWidth": 14,
        "height": 204,
        "marginBottom": -1,
        "maxHeight": 254,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  >
    <ViewManagerAdapter_ExpoVideoView
      accessibilityHint="Video sobre la duración de las reacciones de estrés"
      accessibilityLabel="Video sobre la duración de las reacciones de estrés"
      accessible={true}
      proxiedProperties={
        Object {
          "onError": [Function],
          "onFullscreenUpdate": [Function],
          "onLoad": [Function],
          "onLoadStart": [Function],
          "onReadyForDisplay": [Function],
          "onStatusUpdate": [Function],
          "resizeMode": "mock",
          "source": Object {
            "headers": undefined,
            "overridingExtension": null,
            "uri": "uri",
          },
          "status": Object {
            "rate": 1,
            "shouldPlay": true,
            "volume": 1,
          },
          "useNativeControls": true,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#0077C8",
        "borderBottomColor": "#0077C8",
        "borderBottomWidth": 2,
        "color": "#FFFFFF",
        "fontSize": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "avenir-heavy",
        }
      }
    >
      Las reacciones de estrés pueden interferir. 
    </Text>
    La ayuda profesional es importante para volver a la normalidad.
  </Text>
  <RCTScrollView>
    <View>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Para muchos niños y sus padres las reacciones de estrés traumático temprano mejoran durante el primer mes. Pero aproximadamente 1 de cada 6 sigue teniendo reacciones de estrés traumático que le molestan incluso 6 meses después de una lesión.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Casi todo el mundo se siente perturbado o ansioso al principio. Durante las siguientes semanas y meses, estas reacciones generalmente se debilitan y luego desaparecen. Incluso los niños que se recuperan de lesiones graves comienzan a desarrollar maneras de manejar la situación.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Pero para algunas personas, estas primeras reacciones no parecen mejorar a medida que pasan los meses o incluso pueden empeorar. Aun si se han curado del daño físico, es posible que no se hayan recuperado del trauma. La intensidad y duración de las reacciones de estrés traumático no se relaciona con lo grave que haya sido el daño físico.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        ¿Por qué preocuparse?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Cuando las reacciones de estrés traumático duran semanas o meses, pueden impedir la recuperación de la normalidad y sentirse bien otra vez.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        En niños y adolescentes, esto puede afectar a la escuela, la casa y el juego.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - El trabajo escolar y el aprendizaje pueden verse afectados cuando un niño no puede concentrarse o dormir bien.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Las relaciones familiares y amistades pueden sufrir.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Los niños pueden dejar de hacer cosas que disfrutan o dejar de probar cosas nuevas.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#0077C8",
            "fontFamily": "avenir-heavy",
            "fontSize": 20,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        ¿Qué es el trastorno de estrés postraumático o “TEPT”?
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        El trastorno de estrés postraumático (TEPT) es el nombre dado a las reacciones de estrés traumático que:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Son tan graves que interfieren en la vida normal
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Duran más de un mes.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Es importante saber si su hijo o alguien más de su familia desarrollan el trastorno de estrés postraumático en un accidente o lesión. Es mucho más común de lo que la gente piensa.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        Hasta 1 de cada 10 personas desarrolla trastorno de estrés postraumático en algún momento de su vida.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 16,
            "marginBottom": 3,
            "marginLeft": 5,
            "marginTop": 4,
            "textAlign": "left",
          }
        }
      >
        Después de una lesión, el trastorno de estrés postraumático puede:
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Interferir con la recuperación física
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Contribuir a la aparición de nuevos problemas médicos
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "fontFamily": "avenir-heavy",
            "fontSize": 14,
            "marginHorizontal": 18,
            "marginVertical": 1,
            "textAlign": "left",
          }
        }
      >
        - Simplemente hacer que sea más difícil disfrutar de la vida.
      </Text>
      <Text
        accessibilityHint="Navega a un sitio web sobre las causas del TEPT"
        accessibilityLabel="Navega a un sitio web sobre las causas del TEPT"
        accessible={true}
        onPress={[Function]}
        style={
          Object {
            "alignSelf": "center",
            "color": "#0077C8",
            "fontSize": 16,
            "marginTop": 3,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
            "textDecorationLine": "underline",
          }
        }
        testID="hyperlink"
      >
        Haga clic aquí para obtener más información sobre el TEPT.
      </Text>
      <Text
        style={
          Object {
            "alignSelf": "center",
            "fontSize": 14,
            "marginTop": 5,
            "paddingHorizontal": 5,
            "paddingVertical": 1,
          }
        }
      >
        La buena noticia es que existen muy buenos tratamientos para el TEPT. Desafortunadamente, menos de la mitad de las personas con síntomas de estrés traumático hablan con alguien sobre sus problemas u obtienen ayuda profesional, aun cuando sus síntomas se prolongan durante mucho tiempo.
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
